crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -1,136 +1,180 @@
-              {{ first|truncatechars:20 }}
-            {% endwith %}
-          </a>
-        </td>
-        <td>
-          <a class="td-link display-block" href="{{datum.url}}">
-            {% with email=datum.report.contact_email|default:"—" %}
-              {{ email|truncatechars:20 }}
-            {% endwith %}
-            <br>
-            {% with phone=datum.report.contact_phone|default:"—"  %}
-              {{ phone|truncatechars:20 }}
-            {% endwith %}
-          </a>
-        </td>
-        <td>
-          <a class="td-link display-block" href="{{datum.url}}">
-              {{ datum.report.location_city_town }}, {{ datum.report.location_state }}
-          </a>
-        </td>
-        <td>
-          <a class="td-link display-block" href="{{datum.url}}">
-            {% with summary=datum.report.get_summary %}
-              {{ summary.note|default:"—"|truncatechars:60 }}
-            {% endwith %}
-          </a>
-        </td>
-        <td>
-          <a class="td-link display-block" href="{{datum.url}}">
-            {{ datum.report.last_incident_month|default:"-" }}/{{datum.report.last_incident_day|default:"-"}}/{{ datum.report.last_incident_year|default:"-" }}
-          </a>
-        </td>
-      </tr>
-      {# this row is hidden by default #}
-      <tr id="tr-additional-{{ datum.report.id }}" hidden>
-        <td></td>
-        <td colspan="2">
-          <div class="td-quickview">
-            Contact
-          </div>
-          <div class="display-flex flex-column">
-            <span>
-              {{ datum.report.contact_last_name|default:"-" }} {{ datum.report.contact_first_name|default:"-" }}
-            </span>
-            <span>
-              {{ datum.report.contact_email|default:"-" }}
-            </span>
-            <span>
-              {{ datum.report.contact_phone|default:"-" }}
-            </span>
-          </div>
-        </td>
-        <td colspan="4">
-          <div class="td-quickview">
-            Personal Description
-          </div>
-          <div class="td-summary">
-            {{ datum.report.violation_summary|linebreaks|default:"-" }}
-          </div>
-        </td>
-        <td colspan="2"></td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  {% endif %}
-</table>
-</div>
+<form class="usa-form"
+      method="get"
+      action="{% url 'crt_forms:crt-forms-actions' %}"
+      novalidate
+>
+  <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
+
+  <div class="crt-xscroll">
+    <table class="usa-table crt-table sort-table">
+      <thead>
+        <tr>
+          <th>
+            <div class="usa-checkbox td-checkbox">
+              <input type="checkbox"
+                     name="all"
+                     class="usa-checkbox__input"
+                     value="all"
+                     id="checkbox-all"
+              >
+              <label class="crt-checkbox__label" for="checkbox-all"></label>
```

Comment:
Ooo, this generates an a11y scanning failure I think due to the label itself it empty.

Reply:
Thanks! :man_facepalming: